.tool-versions

```diff
@@ -0,0 +1,2 @@
+nodejs 16.2.0
+golang 1.16.4
```

Comment:
```suggestion
golang latest:1.16
```
This will use the latest version of golang 1.16.x.  I tested this and saw it picked up 1.16.5

Reply:
I might have been hasty agreeing to this, if this file does not change then the docker image wont be updated as the cache wont be invalidated and I dont know the behavior there on run (will it prompt to update?)

You folks have to live with it so whatever works best for you